GenderPayGap.WebUI/Views/AdminOrganisationSector/ChangeSector.cshtml

```diff
@@ -0,0 +1,186 @@
+                This includes:
+                <ul class="govuk-list govuk-list--bullet" style="color: inherit;">
+                    <li>Limited companies</li>
+                    <li>Limited liability partnerships</li>
+                    <li>Charities</li>
+                    <li>Independent and private schools</li>
+                </ul>
+            </text>
+        },
+    };
+
+    var items = new List<ItemViewModel>();
+
+    foreach ((SectorTypes sector, Func<object, object> hint) in sectorHints)
+    {
+        if (Model.Organisation.SectorType != sector)
+        {
+            string fieldName = $"GovUk_Radio_{nameof(AdminChangeSectorViewModel.NewSector)}";
+            string fieldId = $"{fieldName}_{sector}";
+
+            items.Add(new RadioItemViewModel
+            {
+                Name = fieldName,
+                Value = sector.ToString(),
+                Id = fieldId,
+                Label = new LabelViewModel
+                {
+                    Text = sector.ToString(),
+                    For = fieldId
+                },
+                Hint = new HintViewModel
+                {
+                    Html = hint
+                }
+            });
+        }
+    }
+
+}
+
+<span class="govuk-caption-xl">Administration</span>
+<h1 class="govuk-heading-xl">
+    Change sector
+    <br/>
+    <span class="govuk-!-font-size-27">
+        for @(Model.Organisation.OrganisationName)
+    </span>
+</h1>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <form method="POST" action="@Url.Action("ChangeSectorPost", "AdminOrganisationSector", new {id = Model.Organisation.OrganisationId})">
+            @(Html.AntiForgeryToken())
+
+            @(Html.Hidden(nameof(AdminChangeSectorViewModel.Action),
+                ChangeOrganisationSectorViewModelActions.OfferNewSectorAndReason))
+
+            @Html.GovUkErrorSummary()
+
+
+            <table class="govuk-table">
+                <tbody class="govuk-table__body">
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">Organisation name</th>
+                        <td class="govuk-table__cell">@(Model.Organisation.OrganisationName)</td>
+                    </tr>
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">Current sector</th>
+                        <td class="govuk-table__cell">@(Model.Organisation.SectorType)</td>
+                    </tr>
+                </tbody>
+            </table>
+
+            <div class="govuk-form-group">
+                @{
+                    string error = Model.GetErrorFor<AdminChangeSectorViewModel, SectorTypes?>(m => m.NewSector);
+                    ErrorMessageViewModel radiosErrorMessageViewModel = error != null ? new ErrorMessageViewModel {Text = error} : null;
+                }
+                @Html.GovUkRadios(new RadiosViewModel

```

Comment:
Try using `@Html.GovUkRadiosFor`
See how we do this in `Views/AddOrganisationChooseSector/ChooseSector.cshtml`
I think it might make the code a bit neater (both here and in the controller)